command/certificate/format.go

```diff
@@ -13,23 +14,32 @@ import (
 	"github.com/smallstep/cli/ui"
 	"github.com/smallstep/cli/utils"
 	"github.com/urfave/cli"
+
+	"software.sslmate.com/src/go-pkcs12"
 )
 
 func formatCommand() cli.Command {
 	return cli.Command{
-		Name:      "format",
-		Action:    command.ActionFunc(formatAction),
-		Usage:     `reformat certificate`,
-		UsageText: `**step certificate format** <crt_file> [**--out**=<file>]`,
```

Comment:
Let's leave the first arg as `<crt-file>`. That name is consistent across the other commands, and I'm changing the other commands to use `-` instead of `_`.

Reply:
done